src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -642,6 +662,61 @@ ExecuteSingleSelectTask(CitusScanState *scanState, Task *task)
 
 
 /*
+ * BuildPlacementSelectList builds a list of SELECT placement accesses
+ * which can be used to call StartPlacementListConnection or
+ * GetPlacementListConnection.
+ */
+List *
+BuildPlacementSelectList(char *nodeName, int nodePort, List *relationShardList)
```

Comment:
Shouldn't this also be used in GetModifyConnections() / ExecuteSingleModifyTask() / ExecuteSingleSelectTask()?

I'd have commented on those functions, if github would allow to do so... What on earth is the point of not allowing to comment on unchanged pieces of code?